src/lib.rs

```diff
@@ -40,3 +40,4 @@ pub use bundle::Bundle;
 pub use circuit::Proof;
 pub use note::Note;
 pub use tree::Anchor;
+pub use constants::MERKLE_DEPTH_ORCHARD as NOTE_COMMITMENT_TREE_DEPTH;
```

Comment:
For consistency with `sapling-crypto`, we shoudl expose this from `src/tree.rs`.

Reply:
So, in `sapling_crypto` this is also exposed via `pub use` from the root.

Reply:
@str4d should this just be in `tree.rs` or both there and in the root?